README.md

```diff
@@ -35,6 +73,7 @@ Run the tests!
 For background and further discussion, see [A RubyGems Adoption Center](http://www.benjaminfleischer.com/2014/08/17/rubygems-adoption-center/).
 
 ## License
+
```

Comment:
Hi! 

Would it be annoying to ask if you could please remove this newline? Seems out of place :)